error running "pyensembl 75 install"

Haven't attempted to debug this at all, but on dev3 and Python 2.7, I'm getting this:

```
$ pyensembl  75 install

INFO:root:Fetching Homo_sapiens.GRCh37.75 from URL ftp://ftp.ensembl.org/pub/release-75/gtf/homo_sapiens/Homo_sapiens.GRCh37.75.gtf.gz
Downloading ftp://ftp.ensembl.org/pub/release-75/gtf/homo_sapiens/Homo_sapiens.GRCh37.75.gtf.gz to /hpc/users/odonnt02/.cache/ensembl/Homo_sapiens.GRCh37.75
INFO:root:Decompressing gzip into Homo_sapiens.GRCh37.75...
INFO:root:Annotation data for release 75 has just been downloaded
INFO:root:Fetching Homo_sapiens.GRCh37.75.cdna.all.fa from URL ftp://ftp.ensembl.org/pub/release-75/fasta/homo_sapiens/cdna/Homo_sapiens.GRCh37.75.cdna.all.fa.gz
Downloading ftp://ftp.ensembl.org/pub/release-75/fasta/homo_sapiens/cdna/Homo_sapiens.GRCh37.75.cdna.all.fa.gz to /hpc/users/odonnt02/.cache/ensembl/Homo_sapiens.GRCh37.75.cdna.all.fa
INFO:root:Decompressing gzip into Homo_sapiens.GRCh37.75.cdna.all.fa...
INFO:root:Transcript sequence data for release 75 has just been downloaded
Creating database: /hpc/users/odonnt02/.cache/ensembl/Homo_sapiens.GRCh37.75.db
INFO:root:Reading GTF /hpc/users/odonnt02/.cache/ensembl/Homo_sapiens.GRCh37.75 into DataFrame
Traceback (most recent call last):
  File "/demeter/users/odonnt02/sinai/git/ovarian-cancer/projects/pt189/venv-2.7/bin/pyensembl", line 9, in <module>
    load_entry_point('pyensembl==0.5.5', 'console_scripts', 'pyensembl')()
  File "/demeter/users/odonnt02/sinai/git/pyensembl/pyensembl/shell.py", line 45, in run
    data.install()
  File "/demeter/users/odonnt02/sinai/git/pyensembl/pyensembl/ensembl_release.py", line 135, in install
    self._index(force=False)
  File "/demeter/users/odonnt02/sinai/git/pyensembl/pyensembl/ensembl_release.py", line 146, in _index
    if self.db.create(force=force):
  File "/demeter/users/odonnt02/sinai/git/pyensembl/pyensembl/database.py", line 412, in create
    self._create_database(force=force)
  File "/demeter/users/odonnt02/sinai/git/pyensembl/pyensembl/database.py", line 77, in _create_database
    df = self.gtf.dataframe()
  File "/demeter/users/odonnt02/sinai/git/pyensembl/pyensembl/gtf.py", line 221, in dataframe
    self._dataframes[key] = cached_dataframe(csv_path, local_loader_fn)
  File "/demeter/users/odonnt02/sinai/git/pyensembl/pyensembl/compute_cache.py", line 78, in cached_dataframe
    df = compute_fn()
  File "/demeter/users/odonnt02/sinai/git/pyensembl/pyensembl/gtf.py", line 195, in local_loader_fn
    full_df = self._load_full_dataframe()
  File "/demeter/users/odonnt02/sinai/git/pyensembl/pyensembl/gtf.py", line 161, in _load_full_dataframe
    return cached_dataframe(csv_path, self._load_full_dataframe_from_gtf)
  File "/demeter/users/odonnt02/sinai/git/pyensembl/pyensembl/compute_cache.py", line 78, in cached_dataframe
    df = compute_fn()
  File "/demeter/users/odonnt02/sinai/git/pyensembl/pyensembl/gtf.py", line 169, in _load_full_dataframe_from_gtf
    return load_gtf_as_dataframe(path)
  File "/demeter/users/odonnt02/sinai/git/pyensembl/pyensembl/gtf_parsing.py", line 265, in load_gtf_as_dataframe
    df = _read_gtf(filename)
  File "/demeter/users/odonnt02/sinai/git/pyensembl/pyensembl/gtf_parsing.py", line 74, in _read_gtf
    "Must be a GTF file (%s)" % filename
AssertionError: Must be a GTF file (/hpc/users/odonnt02/.cache/ensembl/Homo_sapiens.GRCh37.75)

--- a/pyensembl/gtf.py
+++ b/pyensembl/gtf.py
@@ -37,6 +37,10 @@ class GTF(object):
             species=self.species,
             server=server)
         self.remote_filename = gtf_filename
+        assert self.remote_filename.endswith(".gtf.gz"), \
+            "Expected remote GTF file %s to end with '.gtf.gz'" % (
+                self.remote_filename,)
+
         self.url = join(gtf_url_dir, gtf_filename)
         self.auto_download = auto_download
 
@@ -66,14 +70,12 @@ class GTF(object):
     def local_filename(self):
         """Filename used for local copy of GTF"""
         if self.decompress:
-            return self.base_filename()
+            return self.base_filename() + ".gtf"
         else:
             return self.remote_filename
 
     def local_copy_exists(self):
-        """
-        Has a local copy of the GTF file been downloaded?
-        """
+        """Has a local copy of the GTF file been downloaded?"""
         return self.cache.exists(
             self.url,
             self.local_filename(),

